src/openrct2/peep/Staff.cpp

```diff
@@ -79,6 +79,9 @@ colour_t gStaffHandymanColour;
 colour_t gStaffMechanicColour;
 colour_t gStaffSecurityColour;
 
+// Maximum manhattan distance that litter can be for a handyman to seek to it
+const uint16_t MAX_LITTER_DISTANCE = 0x60;
```

Comment:
```suggestion
const uint16_t MAX_LITTER_DISTANCE = 3 * COORDS_XY_STEP;
```

This makes it clearer that it's 3 tiles (a tile is divided into 32x32 "big" x/y coords).